PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -135,6 +135,51 @@ def nanoAOD_addBoostedTauIds(process, idsToRun=[]):
 
     return process
 
+def nanoAOD_addPNetToTaus(process, addPNetInfo=False, runPNetCHSAK4=False):
+    if addPNetInfo:
+        originalTauName = process.finalTaus.src.value()
+        updatedTauName = originalTauName+'WithPNet'
+        jetCollection = "updatedJets"
+        process.load('RecoBTag.ONNXRuntime.pfParticleNetFromMiniAODAK4_cff')
+        pnetTagName = "pfParticleNetFromMiniAODAK4CHSCentralJetTag"
+        pnetDiscriminators = [];
+        for tag in getattr(process,pnetTagName+"s").flav_names.value():
+            pnetDiscriminators.append(pnetTagName+"s:"+tag)
+
+        # Define "hybridTau" producer
+        from PhysicsTools.PatAlgos.patTauHybridProducer_cfi import patTauHybridProducer
```

Comment:
Is the path to the patTauHybridProducer_cfi.py file correct? It looks like it has not been committed

Reply:
those _cfi are now automatically generated from the `fillDescriptions`, no?

Reply:
It is an auto generated configuration. If you want make an "by eye" inspection of it you should look for it in `$CMSSW_BASE/cfipython/$SCRAM_ARCH/PhysicsTools/PatAlgos` after compilation. Or you can check `PATTauHybridProducer::fillDescriptions` method of `PATTauHybridProducer` in `PhysicsTools/PatAlgos/plugins/PATTauHybridProducer.cc`